ticketingApp/main.js

```diff
@@ -0,0 +1,156 @@
+import './style.css'
+import javascriptLogo from './javascript.svg'
+import viteLogo from '/vite.svg'
+import { setupCounter } from './counter.js'
+
+
+function renderEventTemplate(){
+  const mainContentDiv = document.querySelector('.main-content-component');
+  mainContentDiv.innerHTML = getEventsTemplate();
+  displayCards();
+}
+
+function getEventsTemplate(){
+  return `
+  <header>
+  <nav>
+    <ul class="menu">
+      <li>
+        <a href="index.html">Events</a>
+      </li>  
+      <li>
+        <a href="order.html">Orders</a>
+      </li>
+    </ul>
+  
+  </nav>
+  </header> 
+
+    <footer></footer>`
+}
+
+async function getAllEvents() {
+  const response = await fetch('http://localhost:7042/api/Event/GetAllEvents', {mode:'cors'});
+  const data = await response.json();
+  console.log(response);
```

Comment:
Be careful with logging data to console, not to leak something sensitive. 
You can keep them for now, if you need them for debugging.